lms/templates/instructor/instructor_dashboard_2/certificate-invalidation.underscore

```diff
@@ -3,8 +3,8 @@
 </p>
 
 <div class="add-certificate-invalidation">
-    <input class='student-username-or-email' id="certificate-invalidation-user" type="text" placeholder="<%- gettext('Username or email address') %>" aria-describedby='student-user-name-or-email-tip'>
-    <textarea class='notes-field' id="certificate-invalidation-notes" rows="10" placeholder="<%- gettext('Add notes about this learner') %>" aria-describedby='notes-field-tip'></textarea>
+    <input class='student-username-or-email' id="certificate-invalidation-user" type="text" placeholder="<%- gettext('Username or email address') %>">
```

Comment:
Ah, I see-- https://github.com/edx/edx-platform/pull/12926/files#diff-176ba80b63ee3a096aa50bcd3675e849L7.

Cool, as long as you handle the merges correctly.